src/pages/commits/ActiveCommits/buildVariantCard/WaterfallTaskStatusIcon.tsx

```diff
@@ -62,15 +53,14 @@ export const WaterfallTaskStatusIcon: React.FC<WaterfallTaskStatusIconProps> = (
       trigger={
         <IconWrapper
           onMouseEnter={onHover}
-          onMouseLeave={onUnhover}
           key={`task_${taskId}`}
           aria-label={`${status} icon`}
           to={getTaskRoute(taskId)}
           data-cy="waterfall-task-status-icon"
         >
-          <TaskStatusIconWrapper highlight={shouldHighlight}>
+          <TaskStatusWrapper className={`icon_${identifier}`}>
```

Comment:
I think it's better to use a [data attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*) instead of className. This can prevent unexpected results if styles are ever applied via className for the comparison in `hoverUpdateClassName` .